spec.html

```diff
@@ -36047,6 +36336,8 @@ <h1>
           <dl class="header">
           </dl>
           <emu-alg>
+            1. If _rer_.[[UnicodeSets]] is *true*, then
+              1. Assert: _invert_ is *false*.
```

Comment:
I think this is true, and it's needed for the call to Canonicalize below to be well-typed (as mentioned above).

If it's not true, we'll need a different fix to that line.

```suggestion
              1. Assert: _invert_ is *false*.
              1. Assert: Every element of _A_ consists of a single character.
```

Reply:
_invert_ is only meant to be true for negated character classes of the form `[^…]` in non-UnicodeSets mode:

```
        <emu-grammar>CharacterClass :: `[^` ClassContents `]`</emu-grammar>
        <emu-alg>
          1. Let _A_ be CompileToCharSet of |ClassContents| with argument _rer_.
          1. If _rer_.[[UnicodeSets]] is *true*, then
            1. Return the Record { [[CharSet]]: CharacterComplement(_rer_, _A_), [[Invert]]: *false* }.
          1. Return the Record { [[CharSet]]: _A_, [[Invert]]: *true* }.
        </emu-alg>
```

Reply:
My suggestion didn't have anything to do with `invert` - I was suggesting adding another assertion to make when `[[UnicodeSets]]` is true.